dae/dae/variants_loaders/vcf/tests/test_vcf_loader_multi.py

```diff
@@ -0,0 +1,674 @@
+    #CHROM	POS	    ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	f4.mom	f4.dad	f4.p1	f4.s1	f5.mom	f5.dad	f5.p1	f5.s1	f3.s1
+    1	    865582	.	C	T	.   	.   	EFF=SYN	GT  	1/1 	0/0 	0/1 	0/1 	1/1 	0/0 	1/0 	0/1 	./.
+    1	    865583	.	G	A	.   	.   	EFF=SYN	GT  	0/0 	1/1 	0/1 	0/1 	0/0 	1/1 	1/0 	0/1 	./.
+    1	    865624	.	G	A	.   	.   	EFF=MIS	GT  	1/0 	0/0 	1/0 	0/0 	1/0 	0/0 	1/0 	0/0 	./.
+    1	    865627	.	G	A	.   	.   	EFF=MIS	GT  	0/0 	1/1 	1/0 	0/0 	0/0 	1/1 	1/0 	1/0 	./.
+    1	    865664	.	G	A	.   	.   	EFF=SYN	GT  	0/1 	0/0 	0/1 	0/0 	0/1 	0/0 	0/0 	0/1 	./.
+    1	    865691	.	C	T	.   	.   	EFF=MIS	GT  	1/0 	1/1 	1/0 	1/0 	1/0 	1/1 	0/1 	0/1 	./.
+    """) # noqa
+
+    return vcf_path
+
+
+@pytest.fixture
+def multivcf_ped(tmp_path_factory: pytest.TempPathFactory) -> Path:
+    root_path = tmp_path_factory.mktemp("multivcf")
+    ped_path = setup_pedigree(root_path / "ped_data" / "in.ped", """
+    familyId	personId	dadId	momId	sex	status	role	phenotype
+    f1	        f1.mom  	0   	0	    2	1	    mom 	unaffected
+    f1	        f1.dad  	0	    0	    1	1	    dad 	unaffected
+    f1	        f1.p1   	f1.dad	f1.mom	1	2	    prb 	autism
+    f1	        f1.s1   	f1.dad	f1.mom	2	2	    sib	    autism
+    f2	        f2.mom  	0	    0	    2	1	    mom 	unaffected
+    f2	        f2.dad  	0	    0	    1	1	    dad 	unaffected
+    f2	        f2.p1   	f2.dad	f2.mom	1	2	    prb	    autism
+    f2	        f2.s1   	f2.dad	f2.mom	2	2	    sib	    autism
+    f3	        f3.mom  	0	    0	    2	1	    mom 	unaffected
+    f3	        f3.dad  	0	    0	    1	1	    dad 	unaffected
+    f3	        f3.p1   	f3.dad	f3.mom	1	2	    prb	    autism
+    f3	        f3.s1   	f3.dad	f3.mom	2	2	    sib	    autism
+    f4	        f4.mom  	0	    0	    2	1	    mom 	unaffected
+    f4	        f4.dad  	0	    0	    1	1	    dad 	unaffected
+    f4	        f4.p1   	f4.dad	f4.mom	1	2	    prb 	autism
+    f4	        f4.s1   	f4.dad	f4.mom	2	2	    sib 	autism
+    f5	        f5.mom  	0	    0	    2	1	    mom 	unaffected
+    f5	        f5.dad  	0	    0	    1	1	    dad 	unaffected
+    f5	        f5.p1   	f5.dad	f5.mom	1	2	    prb 	autism
+    f5	        f5.s1   	f5.dad	f5.mom	2	2	    sib	    autism
+    """)
+
+    return ped_path
+
+
+@pytest.fixture
+def simple_vcf_loader(
+    gpf_instance: GPFInstance
+) -> Callable[[Path, Path, Any], VcfLoader]:
+    def _split_all_ext(filename: str) -> str:
+        res, ext = os.path.splitext(filename)
+        while len(ext) > 0:
+            res, ext = os.path.splitext(res)
+        return res
+
+    def ctor(ped: Path, vcf: Path, additional_params: Any) -> VcfLoader:
+        ped_filename = _split_all_ext(str(ped)) + ".ped"
+        families_loader = FamiliesLoader(ped_filename)
+        families = families_loader.load()
+        params = additional_params
+        vcf_filename = vcf
+
+        return VcfLoader(
+            families, [str(vcf_filename)],
+            genome=gpf_instance.reference_genome, params=params,
+        )
+    return ctor
```

Comment:
Move this fixture closer to the tests that use it. Consider moving `add_chrom_prefix`, `del_chrom_prefix` tests into a separate file.